org-timeline.el

```diff
@@ -168,9 +172,11 @@ Return new copy of STRING."
   "Insert graphical timeline into agenda buffer."
   (unless (buffer-narrowed-p)
     (goto-char (point-min))
-    (while (and (eq (get-text-property (line-beginning-position) 'org-agenda-type) 'agenda)
-                (not (eobp)))
-      (forward-line))
+    (if org-timeline-prepend 
```

Comment:
This can be written more idiomatically as `(unless org-timeline-prepend (while...))`